app/src/lib/editors/win32.ts

```diff
@@ -20,25 +20,25 @@ type RegistryKey = { key: HKEY; subKey: string }
 
 type WindowsExternalEditorPathInfo =
   | {
-      /**
-       * Registry key with the install location of the app. If not provided,
-       * 'InstallLocation' will be used.
-       **/
-      readonly installLocationRegistryKey?: 'InstallLocation'
-
-      /**
-       * List of lists of path components from the editor's installation folder to
-       * the potential executable shims. Only needed when the install location
-       * registry key is `InstallLocation`.
-       **/
-      readonly executableShimPaths: ReadonlyArray<ReadonlyArray<string>>
-    }
+    /**
+    * Registry key with the install location of the app. If not provided,
+    * 'InstallLocation' will be used.
+    **/
+    readonly installLocationRegistryKey?: 'InstallLocation'
+
+    /**
+    * List of lists of path components from the editor's installation folder to
+    * the potential executable shims. Only needed when the install location
+    * registry key is `InstallLocation`.
+    **/
+    readonly executableShimPaths: ReadonlyArray<ReadonlyArray<string>>
+  }
   | {
-      /**
-       * Registry key with the install location of the app.
-       **/
-      readonly installLocationRegistryKey: 'DisplayIcon'
-    }
+    /**
+     * Registry key with the install location of the app.
+     **/
+    readonly installLocationRegistryKey: 'DisplayIcon'
+  }
```

Comment:
These whitespace changes are incorrect, sorry, you must follow the same indentation rules as us or the linter won't allow building this.

Reply:
I know, but when I press Ctrl+S in VS Code it automatically changes the whitespaces

Reply:
I am not a typescript developer so it may be my problem, I am gonna try to fix it.